.golangci.yml

```diff
@@ -58,7 +58,8 @@ linters:
     - deadcode
     - depguard
     - dogsled
-    - dupl
+    # TODO
+    #- dupl
```

Comment:
when Moshe said to ignore the check in the code i believe he meant to add above the problematic method:
`//nolint:dupl`

see existing code: https://github.com/Mellanox/network-operator/blob/a24d16b212d3b2bad2d5df04018f9be722216b8f/pkg/state/state_nv_peer.go#L59

Reply:
I pushed one, but I overwritten it by mistake
https://github.com/Mellanox/network-operator/compare/a876e87ef9e4b7400d1a84e277bfeb7d8d7a2a91..455ff0dc60c9dc35701a77718ca42ca1008211ec

readded